Xamarin.Forms.Platform.Android/VisualElementPackager.cs

```diff
@@ -159,7 +160,7 @@ void OnChildAdded(object sender, ElementEventArgs e)
 			if (view != null)
 				AddChild(view);
 
-			if (ElementController.LogicalChildren[ElementController.LogicalChildren.Count - 1] != view)
+			if (ElementController.LogicalChildren.FirstOrDefault() != view)
```

Comment:
we try not to use linq expressions

Reply:
I understand. But in the case it is correct. Safety is better.

Reply:
Why not **Last**orDefault?

Reply:
You are correct. It should be last or default.

Reply:
well, that will enumerate all the children for nothing. probably why we used the indexer before...

Reply:
Actually, did not enumerate all children, If it's instance of IList